Codemagic CodePush CLI tools (#3143)
codemagic-ci-cd/codemagic-docs: content/integrations/codepush-integration.md

{{</notebox>}}

{{<notebox>}}
**Note**: Running **code-push release-react** generates updates and releases them to the server to be served 
{{</notebox>}}

7. In order to reveal the Deployment keys, run **code-push-standalone deployment ls YOUR_APP_NAME -k**
8. By default, you get two Deployment channels: Staging and Production. You can add new ones, rename or delete them by running the following commands:

{{< highlight bash "style=paraiso-dark">}}
To Add: code-push deployment add <appName> <deploymentName>
To Remove: code-push deployment rm <appName> <deploymentName>
To Rename: code-push deployment rename <appName> <deploymentName> <newDeploymentName>
{{< /highlight >}}

9. Likewise, apps can be added, renamed and deleted:

{{< highlight bash "style=paraiso-dark">}}
To Add: code-push app add <appName>
To Rename: code-push app rename <appName> <newAppName>
To Delete: code-push app rm <appName>
{{< /highlight >}}

> 10. If you need to patch releases e.g. you need to make a change in a previous release e.g. increase rollout percentage, a missed bug fix etc. you can achieve it by running **code-push patch <appName> <deploymentName>**

Reviewer: ```suggestion
10. If you need to patch a previous release, e.g. increase the rollout percentage, push a missed bug fix or similar, you can achieve it by running **code-push patch <appName> <deploymentName>**
```

> 11. You cannot delete a deployment release history but you can roll it back in case any release was shipped with a broken feature or anything, by running **code-push rollback <appName> <deploymentName>**

Reviewer: I'm not entirely sure what the correct idea is here -- is it that you cannot delete a deployed release or you can't delete history?

```suggestion
11. You cannot delete a deployment release history but you can roll back a release by running **code-push rollback <appName> <deploymentName>**
```

Author: once deployed a release, you cannot delete its data, you can only roll it back

Author: that'd do it
